06-working-w-copilot/setting-up-copilot.md

```diff
@@ -0,0 +1,84 @@
+
+2. Go to the account settings screen by clicking your profile photo in the upper-right corner of any page, then click "Settings". Under the "Code, planning, and automation" section of the "Settings" sidebar, click "Copilot".  
+   ![The GitHub "Settings" menu with the "Code, planning, and automation" section underlined and the "Copilot" option circled](assets/setting-up-copilot/github-settings-menu.png)  
+   *Fig. "Copilot" option in the "Settings" menu*
+
+3. Click “Enable Copilot” and we should see a message that we have access to Copilot for free and a new button “Continue to get access to Github Copilot” that we should click.
+
+4. We will be taken to a preferences screen where we can choose some options about how copilot will use public code and snippets it generates. 
+   - Choose what you are most comfortable with, these choices can be updated at any time by navigating to "Settings" > "Copilot".
+
+## Setting up Copilot in VS Code
+
+We will be using Copilot exclusively from VS Code in this lesson. Feel free to follow your curiosity and look up if plugins are available for any other IDEs you would like to try Copilot out in. 
+
+To add Copilot to VS Code, we need to install the Copilot extension:
+
+1. Open VS Code, then click the Extensions tab.
+ 
+2. In the Extensions tab, search for "GitHub Copilot"  
+   ![The "Extensions" tab of VS Code with the search terms "GitHub Copilot" entered in the search bar. The results show "GitHub Copilot" as the first result, which is circled in red.](assets/setting-up-copilot/vscode-search-copilot.png)  
+   *Fig. "GitHub Copilot" in the VS Code Extension search results*
+
+3. You can install GitHub Copilot using the "Install" button directly on the entry in the results list or click the "GitHub Copilot" result to go to their extension page and click "Install" from there.  
+   ![The information page for the GitHub Copilot Extension](assets/setting-up-copilot/vscode-copilot-extension-page.png)  
+   *Fig. "GitHub Copilot"'s extension page in VS Code*
+
+4. If you haven't previously authorized VS Code in your GitHub account, you will be asked to sign in to GitHub inside VS Code (If you've previously authorized Visual Studio Code in your GitHub account, GitHub Copilot should automatically be authorized).
+   - In your browser, GitHub will request the necessary permissions for GitHub Copilot. To approve these permissions, click Authorize Visual Studio Code.
+   - In Visual Studio Code, in the "Visual Studio Code" dialogue box, click "Open" to confirm the authentication.
+
+If you'd like to see instructions for setting up the Copilot extension in VS Code written another way, check out GitHub's ["Getting started with GitHub Copilot"](https://docs.github.com/en/copilot/using-github-copilot/getting-started-with-github-copilot#installing-the-github-copilot-extension-in-visual-studio-code) documentation.
```

Comment:
- Since we're back out of steps, this can be phrased to avoid you voicing.

```suggestion
For instructions to set up the Copilot extension in VS Code written another way, check out GitHub's ["Getting started with GitHub Copilot"](https://docs.github.com/en/copilot/using-github-copilot/getting-started-with-github-copilot#installing-the-github-copilot-extension-in-visual-studio-code) documentation.
```